usagov/vote-gov-drupal
# VOTE-3022 Update NVRF app to v0.20.0

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-3022

## Description

Update NVRF app to [v0.20.0](https://github.com/usagov/vote-gov-nvrf-app/releases/tag/v0.20.0)

## Deployment and testing

### Post-deploy steps

1. cd into vote_nvrf module, npm run build
2. lando retune

### QA/Testing instructions

1. Confirm changes in the changelog of v0.20.0 are applied to the NVRF app.
2. Test all fields in the NVRF app in English and Spanish, and confirm the data accurately displays in the final PDF.

## Checklist for the Developer




## Checklist for the Peer Reviewers
